src/Ticket.php

```diff
@@ -4609,9 +4624,22 @@ public static function showCentralList($start, $status = "process", bool $showgr
                         $options['criteria'][0]['value']      = CommonITILValidation::WAITING;
                         $options['criteria'][0]['link']       = 'AND';
 
-                        $options['criteria'][1]['field']      = 59; // validation aprobator
-                        $options['criteria'][1]['searchtype'] = 'equals';
-                        $options['criteria'][1]['value']      = Session::getLoginUserID();
+                        $options['criteria'][1]['criteria'][0]['field']      = 59; // validation aprobator user
+                        $options['criteria'][1]['criteria'][0]['searchtype'] = 'equals';
+                        $options['criteria'][1]['criteria'][0]['value']      = 'myself'; // Resolved as current user's ID
+                        $options['criteria'][1]['criteria'][0]['link']       = 'AND';
```

Comment:
Should it be 'OR' ?
```suggestion
                        $options['criteria'][1]['criteria'][0]['link']       = 'OR';
```

Reply:
This is counter intuitive : field search type and value are the 3 values of a simple expression and link is the logical operator linking several expressions together. 

Link is usually placed after the simple expression but is handled before the simple expression. As a boolean operator is meaningless at the beginning of a complex expression and it is ignored. Actually we can just remove the line.

Same for the 'OR' a few lines below by the way.